robyn/openapi.py

```diff
@@ -197,18 +211,30 @@ def get_path_obj(self, endpoint: str, name: str, description: str, tags: List[st
         @param description: Optional[str] short description of the endpoint (to be fetched from the endpoint defenition by default)
         @param tags: List[str] for grouping of endpoints
         @param query_params: Optional[TypedDict] query params for the function
+        @param request_body: Optional[TypedDict] request body for the function
         @param return_type: str return type of the endpoint handler
 
         @return: (str, dict) a tuple containing the endpoint with path params wrapped in braces and the "path" openapi object
         according to spec
         """
+
+        if not description:
+            description = "No description provided"
+
+        openapi_path_object = {
+            "summary": name,
+            "description": description,
+            "parameters": [],
```

Comment:
what does this field represent?

Reply:
We are appending it below. This line is for init. Else, it complained "dict doesn't contain key parameter", IIRC. I'll check.